plugins/discovery-file/build.gradle

```diff
@@ -38,7 +38,7 @@ task setupSeedNodeAndUnicastHostsFile(type: DefaultTask) {
 // setup the initial cluster with one node that will serve as the seed node
 // for unicast discovery
 ClusterConfiguration config = new ClusterConfiguration(project)
-config.distribution = 'integ-test-zip'
+config.distribution = System.getProperty('tests.distribution', 'integ-test-zip')
```

Comment:
I'm not sure we need this line at all given your change to `PluginBuildPlugin`.

Reply:
This would take precedence and override that. So it would be set back to `integ-test-zip` regardless of the property.

Reply:
I am concerned this is fragile. I don't want to see this logic copied around to other projects. Can you please create an issue for making creating additional cluster fixtures easier, so that the default can be set by PluginBuildPlugin for this project and reused the we create this cluster?

Reply:
++ on this being fragile. Fixing it in a followup is fine with me though.